po/ru.ebook-speaker.po

```diff
@@ -0,0 +1,355 @@
+msgid "G               - go to page number"
+msgstr "G                - перейти на страницу"
+
+#: src/ebook-speaker.c:830 src/list_dir.c:187
+msgid "h,?             - give this help"
+msgstr "h,?           -показать данную справку"
+
+#: src/ebook-speaker.c:832
+msgid "j,5             - just play current item"
+msgstr "j,5                      - проиграть текущий элемент"
+
+#: src/ebook-speaker.c:834
+msgid "l               - switch to next level"
+msgstr "l               - перейти на следующий уровень"
+
+#: src/ebook-speaker.c:836
+msgid "L               - switch to previous level"
+msgstr "L             - перейти на предыдущий уровень"
+
+#: src/ebook-speaker.c:838
+msgid "m               - mute sound output on/off"
+msgstr "m              - включение/выключение приглушения звука"
+
+#: src/ebook-speaker.c:840 src/list_dir.c:193
+msgid "n               - search forwards"
+msgstr "n           - искать вперёд"
+
+#: src/ebook-speaker.c:842 src/list_dir.c:195
+msgid "N               - search backwards"
+msgstr "N            - искать в обратном направлении"
+
+#: src/ebook-speaker.c:844
+msgid "o               - select next output sound device"
+msgstr "o           - выбрать следующее устройство воспроизведения"
+
+#: src/ebook-speaker.c:846
+msgid "p               - place a bookmark"
+msgstr "p               - установить закладку"
+
+#: src/ebook-speaker.c:855
+msgid "q               - quit eBook-speaker and place a bookmark"
+msgstr "q               - завершить работу ebook-speaker и установить закладку"
+
+#: src/ebook-speaker.c:858
+msgid "r               - rotate scanned document"
+msgstr "r             - повернуть отсканированный документ"
+
+#: src/ebook-speaker.c:860
+msgid "s               - stop playing"
+msgstr "s             - остановить воспроизведение"
+
+#: src/ebook-speaker.c:862
+msgid "S               - show progress on/off"
+msgstr "S        - показ прогресса включен/выключен"
+
+#: src/ebook-speaker.c:864
+msgid "t               - select next TTS"
+msgstr "t               - выбрать следующий синтезатор речи"
+
+#: src/ebook-speaker.c:866
+msgid "T               - move cursor to the first item"
+msgstr "T             - переместить курсор к первому элементу"
+
+#: src/ebook-speaker.c:868
+msgid "U,+             - increase playing speed"
+msgstr "U,+              - увеличить скорость воспроизведения"
+
+#: src/ebook-speaker.c:870
+msgid "v,1             - decrease playback volume"
+msgstr "v,1             - уменьшить громкость воспроизведения"
+
+#: src/ebook-speaker.c:872
+msgid "V,7             - increase playback volume (beware of Clipping)"
+msgstr "V,7           - увеличить громкость воспроизведения (учитывайте перегрузку)"
+
+#: src/ebook-speaker.c:874
+msgid "w               - store whole book to disk in WAV-format"
+msgstr "w          - сохранить книгу на диск в WAV формате"
+
+#: src/ebook-speaker.c:876
+msgid "x               - go to the file-manager"
+msgstr "x           - открыть файловый менеджер"
```

Comment:
Please take care of the alignment of the items. These shows up in the help page, so we'd rather have the various items properly aligned.

Reply:
What do you mean by alignment? Is it the mesurement/amount of spaces after key combinations? Thank you.

Reply:
Yes. The total amount of characters should be less than the usual 80 of a terminal, and the amount of characters before the dash should be coherent so that the dashes get aligned.